src/main/common/filter.c

```diff
@@ -29,7 +29,7 @@
 #include "common/maths.h"
 #include "common/utils.h"
 
-#define BIQUAD_Q 1.0f / sqrtf(2.0f)     /* quality factor - 2nd order butterworth*/
+#define BIQUAD_Q 1.0f / M_SQRT2f     /* quality factor - 2nd order butterworth*/
```

Comment:
This is early optimisation - the whole const expression will be pre-calculated by the compiler. In reality `M_SQRT2f` should probably be removed from `maths.h`, as there is no reason to use it over `sqrtf(2.0f)` with a modern compiler.

Reply:
Is there an analogous constexpr that would obviate this, or am I being dumb?